src/main/java/com/google/api/codegen/transformer/py/PythonGapicSurfaceTransformer.java

```diff
@@ -227,24 +234,30 @@ private ViewModel generateApiClass(GapicInterfaceContext context) {
         namer.getAndSaveNicknameForGrpcClientTypeName(
             context.getImportTypeTable(), context.getInterfaceModel()));
 
-    xapiClass.apiMethods(methods);
-
     xapiClass.toolkitVersion(GeneratorVersionProvider.getGeneratorVersion());
     xapiClass.gapicPackageName(
         namer.getGapicPackageName(packageConfig.packageName(TargetLanguage.PYTHON)));
     xapiClass.fileHeader(fileHeaderTransformer.generateFileHeader(context));
 
+    // Generate the view for the API class.
+    xapiClass.templateFileName(XAPI_TEMPLATE_FILENAME);
+    xapiClass.outputPath(namer.getSourceFilePath(subPath, name));
+    xapiClass.name(name);
+    xapiClass.apiMethods(methods.stream().collect(Collectors.toList()));
+
     return xapiClass.build();
   }
 
-  private List<ApiMethodView> generateApiMethods(GapicInterfaceContext context) {
-    ImmutableList.Builder<ApiMethodView> apiMethods = ImmutableList.builder();
+  private List<ApiMethodView> generateApiMethodsAndSamples(GapicInterfaceContext context) {
```

Comment:
Shouldn't this just be `generateApiMethods`?

Reply:
Actually, this is cruft that is no longer used. Removed.